ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/route-maps.j2

```diff
@@ -1,22 +1,16 @@
 {# eos - route-maps #}
-{% if route_maps is defined and route_maps is not none %}
-{%     for route_map in route_maps | arista.avd.natural_sort %}
-{%         for sequence in route_maps[route_map].sequence_numbers | arista.avd.natural_sort %}
+{% for route_map in route_maps | arista.avd.natural_sort %}
```

Comment:
keep "if route map is arista.avd.defined" for files consistency

Reply:
I disagree. We should not have redundant code in the templates unless it adds to readability. We included the variable checks in arista.avd.natural_sort to avoid this extra check before the loops.

Reply:
I agree